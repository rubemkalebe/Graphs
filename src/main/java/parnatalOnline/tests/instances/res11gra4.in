11 4
24 24 5 9 7 28 21 5 23 14
13 8 11 28 30 24 18 3 7
2 23 10 5 15 8 27 8
9 13 25 26 14 22 23
10 20 15 22 11 7
10 3 21 26 28
6 9 2 21
24 3 12
13 30
3
