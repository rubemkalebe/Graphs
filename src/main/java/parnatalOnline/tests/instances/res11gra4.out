38.000000
0 3, 0 8, 1 9, 2 3, 2 6, 4 10, 5 7, 6 9, 7 9, 9 10, 
2204848800
3509.961653;0.056016;